components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/APIDefinitionFromSwagger20.java

```diff
@@ -236,8 +236,8 @@ public String getAPIDefinition(APIIdentifier apiIdentifier, Registry registry) t
      */
     @Override
     @SuppressWarnings("unchecked")
-    public String createAPIDefinition(API api) throws APIManagementException {
-        JSONParser parser = new JSONParser();
+    public String generateAPIDefinition(API api) throws APIManagementException {
+        /*JSONParser parser = new JSONParser();
```

Comment:
Any reason to keep commented code? If previous code is needed it can be fetched from git history. It might better to clean this.